README.txt

```diff
@@ -0,0 +1,72 @@
+This branch is the github pages repo for armadaproject.io. This document
+contains tips and conventions about how to develop the website interactively.
+
+
+Conventions
+-----------
+
+Integration with master branch
+==============================
+The Armada master branch contains github workflows that will copy over some
+files when committed to that branch. Ensure no changes you are making here
+will be overwritten on the next master push, and make changes in the master
+branch when appropriate.
+
+Interesting Files/Directories
+=============================
+- All markdown files in the root of the branch are rendered as webpages, by
+  default at the path `/blah` for a given file `blah.md` (but this behavior
+  can be overridden by jekyll headers).
+- `_pages/` is the pages collection, as defined in `_config.yml` -- generally
+  speaking, it's best to leave the markdown files in the root directory
+  instead unless you have a specific reason. 
+- `_layouts/` contains layouts which can be used to format markdown files.
+  - `default.html` is the default layout, used by almost all pages.
+  - `redirect.html` is a layout designed to 301 redirect based on headers.
+- `_includes/` includes various files included to build the page. such as
+  headers, footers, and similar.
+- `_data/settings.yml` is an additional settings file, which contains the
+  tree used to render the navigation bar.
+
+
+Testing
+-------
+armadaproject.io is a jekyll generated site, using github pages. You have two
+choices for running it locally for testing.
+
+Local testing
+=============
+
+Prerequisites
+^^^^^^^^^^^^^
+- Ensure you have ruby 2.7 and bundler available.
```

Comment:
Can you spell out what you mean by bundler?  I have never developed with ruby.

Reply:
It's the command referenced below, `bundle`. It's kinda like ruby's version of `pip` and `virtualenv` sorta wrapped up in one.

Reply:
Hm, I think I'm just going to link to https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/testing-your-github-pages-site-locally-with-jekyll for any questions; it's basically the bigger version of what I have posted there.